.github/workflows/coverage.yml

```diff
@@ -0,0 +1,28 @@
+name: Test Coverage
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      
+      - name: Test Coverage
+        run: make coverage
+        
+      - name: Setup LCOV
+        uses: hrishikesh-kadam/setup-lcov@v1
```

Comment:
ultimately up to you, but for our other projects, I don't use versions because they are mutable vs the commit itself, except for anything in the `actions/` space. 

See https://github.com/macadmins/osquery-extension/blob/main/.github/workflows/build_macadmins_extension.yml